benchmarks/run.js

```diff
@@ -0,0 +1,104 @@
+'use strict'
```

Comment:
I would add a script in package json:
`bench: "node benchmarks/run.js"`

Reply:
Great idea, i'll add that!